std/file.d

```diff
@@ -2983,6 +2983,31 @@ auto dirEntries(string path, SpanMode mode, bool followSymlink = true)
     return DirIterator(path, mode, followSymlink);
 }
 
+/// Duplicate functionality of D1's $(D std.file.listdir()):
+unittest
+{
+  string[] listdir(string pathname)
```

Comment:
All I'm saying is that we need to be consistent in the coding style we present in official documentation. Personally, I don't care how the code is formatted -- but it greatly distracts from the docs if it's not consistently applied. Imagine a newcomer browsing over dlang.org, and finding that one page uses 2-space indentation, the next page uses 4-space indentation, and some obscure bitrotted page uses 3-space indentation. He will instantly think, "what a bunch of unprofessional amateurs, I better go look at a more well-developed language". That's why I said, if we decide now that we should use 2-space indentation for all code examples, then we need to be consistent across the board -- language docs, Phobos unittest examples, *everything*. I don't like it that this PR introduces a coding style that's different from the rest. It is detracting from the overall D brand. I don't care whether we decide on 2-space indentation or 4-space indentation, but it should not be on an ad hoc basis. Either we do it -- for *all* documentation across the board -- or we don't -- including in this PR.